js/csvParser.js

```diff
@@ -19,21 +37,25 @@ export function combineCSV(vaultCSV, gmcCSV) {
 
     // 2. I would then need to compare the GMC asset tag against
     // the vault asset tag and write a note if it doesn't match
+    // TODO(@Nyrvous): it would be good to add a helpful note here
     let note;
     if (gmcRow.annotatedAssetId !== vaultRow.assigned_asset_tag) {
       note = `Why you not matching? Or a better note? ${gmcRow.annotatedAssetId} - ${vaultRow.assigned_asset_tag}`;
     }
```

Comment:
@Nyrvous It is nice to discuss code on Github. You can add comments to any changed line. Call people out on bad code, link to reference material or even add emojis or gifs 😃